pixeltable/catalog/table.py

```diff
@@ -1174,29 +1175,39 @@ def _drop_index(
             idx_id = idx_info[0].id
         self._tbl_version.get().drop_index(idx_id)
 
+    if TYPE_CHECKING:
```

Comment:
Move to top of file. (General comment: Preferred Python style is to always put `import` statements at the top of the file unless there's a specific reason why they can't go there. And I don't think such a reason can ever exist for imports that are protected by `if TYPE_CHECKING`.)